src/db-updates.php

```diff
@@ -2296,6 +2296,41 @@ function __try($sql, $cb = null){
         __exec("UPDATE opportunity set registration_ranges = '[]' WHERE registration_ranges IS null OR registration_ranges::VARCHAR = '\"\"'");
         __exec("UPDATE opportunity set registration_categories = '[]' WHERE registration_categories IS null OR registration_categories::VARCHAR = '\"\"'");
     },
+    'Atualizando as áreas de Atuação' => function() use($conn) {
+        __try("
+        -- Adicionando áreas novas
+        with areas(term) as(
+            values ('Acervos Públicos ou de Interesse Público'), ('Animação'), ('Áreas técnicas da cultura'), ('Arte-Educação'), ('Artes Cênicas'), ('Arte Gráfica'), ('Arte Santeira'), ('Artes Clássicas'), ('Artes do Espetáculo'), ('Artes Integradas'), ('Bandas de Congo e Ticumbi'), ('Bibliotecas'), ('Caboclinho'), ('Canto Coral'), ('Capoeira'), ('Carimbó'), ('Centros de Memória'), ('Carnaval'), ('Cavalo Marinho'), ('Ciranda'), ('Congado'), ('Conservação e Restauro'), ('Construção Civil Tradicional'), ('Cultura Afro-Brasileira'), ('Cultura Alimentar'), ('Cultura DEF'), ('Cultura e Direitos Humanos'), ('Cultura Urbana'), ('Cultura Hip-Hop e Funk'), ('Cultura, Infância e Adolescência'), ('Culturas dos Povos de Comunidades Tradicionais de Matriz Africana'), ('Culturas dos Povos Nômades'), ('Culturas Populares e Tradicionais'), ('Culturas de Povos de Terreiros'), ('Culturas Quilombolas'), ('Dança de Salão'), ('Desenho Industrial'), ('Design de Interiores'), ('Design Paisagístico'), ('Escultura'), ('Economia da Cultura'), ('Tradição Oral e Falares'), ('Festas Populares'), ('Festejos Juninos'), ('Feiras e Exposições'), ('Forró'), ('Grafite'), ('Gravura'), ('Humor'), ('Intercâmbio Cultural'), ('Internet Podcasting'), ('Jongo'), ('Jornais e Revistas'), ('Lidas Campeiras'), ('Linguagem'), ('Línguas'), ('Manifestações Artísticas Culturais Afro-Brasileiras'), ('Maracatu'), ('Memória'), ('Mídias Livres'), ('Mímica'), ('Música Erudita'), ('Música Instrumental'), ('Música Popular'), ('Ourivesaria'), ('Paisagens culturais'), ('Paradista'), ('Patrimônio Natural'), ('Performance'), ('Pesca Artesanal'), ('Pesquisa & Desenvolvimento (P&D)'), ('Pintura'), ('Pintura Corporal'), ('Produção de Doces Tradicionais'), ('Produção de Eventos'), ('Quadrilha Junina'), ('Reisado'), ('Sítios Históricos e Arqueológicos'), ('Tambor de Crioula'), ('Tecnologias Culturais'), ('Tradições'), ('Vídeo'), ('Xondaro')
+        )
+        insert into term (id, taxonomy, term)
+         select nextval('term_id_seq'::regclass),
+                'area' as taxonomy,
+                a.term as term
+           from areas a
+      left join term t on t.term = a.term
+          where t.id is null;");
+
+        __exec("update term set term = 'Arquitetura e Urbanismo' where id = 270;");
```

Comment:
Preservar padrão do código utilizando palavras reservadas em caixa alta. Por exemplo:
```suggestion
        __exec("UPDATE term SET term = 'Arquitetura e Urbanismo' WHERE id = 270;");
```